tangelo/linq/circuit.py

```diff
@@ -440,6 +440,13 @@ def finalize_cmeasure_control(self):
         if isinstance(self._cmeasure_control, ClassicalControl):
             self._cmeasure_control.finalize()
 
+    def fix_variational_parameters(self):
+        """Fixes all variational parameters in this circuit, making the corresponding gates non-variational."""
```

Comment:
Fix

Reply:
Fixed in [commit 37a9497](https://github.com/goodchemistryco/Tangelo/pull/398/commits/37a94975fdc86946aeb7c4a7e5a0f9a7555aa9b7).